### PR TITLE
Add recommended tv show details

### LINKED ISSUE #22: Add recommendation file for tv show
# Rick and Morty Season 3
 - [x] Switch your channel to Adult Swim
 - [x] Search for air time
 - [x] Buy some Cheetos
 - [x] Watch the show

### PR DESCRIPTION
**Pull request by rsjiezhang**
This is to resolve #22.

### CODE REVIEW

**Comment by rsjiezhang** [comment]
What is the way to merge multiple columns within a table with markdown? 
colspan starts from the second column.

**Comment by zye4252264** [comment]
Test work

**Comment by zye4252264** [comment]
:100:

**Comment by TinaJin-zz** [comment]
hi 🎉

File: rickandmortys3.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
...icks and kill the others before going to the Citadel of Ricks. Once transferred into a Rick with higher rank, Rick manages to transport the Citadel in the same space as the Federation prison, prompting a massive battle. Rick uses the confusion to access the Federation mainframe—revealed to be his reason for turning himself in— allowing him to make the galactic currency worthless therefore destroying the Federation's economy. The Federation falls into chaos as a result, and leaves Earth. Rick returns to the Smith household, but Jerry issues Beth an ultimatum: choose him or Rick. Beth announces she is leaving Jerry. The status quo restored, Rick reveals to Morty his ulterior motive was to make both the Federation and Jerry "go away", making him Morty's de facto male influence. This devolves into Rick going on a nonsensical angry rant, in homage to his rant in the pilot episode, about how their adventures are about to get darker than before and how his ultimate goal is to get himself some McDonald's Szechuan sauce. <br> Post-Credit Scene: Tammy observes Bird Person's resurrection as a cyborg dubbed "Phoenix Person". <br> Guest starring: Nathan Fillion as The Galactic Federation Agent.
```
Comment (TinaJin-zz) [inline rickandmortys3.md]: nice

**Review by TinaJin-zz — COMMENTED** [review]
👍

File: rickandmortys3.md (hunk 1 above)
Comment (rsjiezhang) [reply rickandmortys3.md]: thanks

**Comment by rsjiezhang** [comment]
@TinaJin add tv show name, please review and approve.

**Review by TinaJin-zz — COMMENTED** [review]
👍

**Comment by rsjiezhang** [comment]
@TinaJin under the review change drop down, there are three options, one of them is "Approve"

**Review by TinaJin-zz — CHANGES_REQUESTED** [review]
looks nice, ready to merge!

**Review by hmassih — COMMENTED** [review]
This looks like a good TV show - thanks for the recommendation.

**Comment by rsjiezhang** [comment]
@hmassih yeah, it was super funny.

**Review by TinaJin-zz — APPROVED** [review]

**Pull request merged**